### DISCUSSION TITLE
[Feature Suggestion] Automatic Cloud Sync

### DISCUSSION THREAD

**Discussion by Tormentor667** [351ELEC/351ELEC#860] [Ideas]
Dear developers,

I have three different Anbernic devices: One at home, one at work and one at my girlfriend's home. I am using the syncing tool to make sure that my savegames are the same on each device. Though sometimes I simply forget to upload or download my latest savegames on one device's location and then being on the other one I can't continue playing because of that issue.

Would it be possible to include the option to

- automatically download (restore) the cloud save files at AmberElec's startup
- automatically upload (save) the local save files to the cloud when exiting a game

Best regards,
Torm

**Comment by MrScottyTay** [comment]
can we please get this, ArkOS does this through [Arklone](https://github.com/ridgekuhn/arklone-arkos#settings-dialog) surely AmberELEC can do something similar
